frontend/components/package.json

```diff
@@ -23,10 +23,9 @@
   },
   "scripts": {
     "build": "webpack --progress --hide-modules --config  webpack.build.js",
-    "builddev": "webpack --progress --hide-modules --config  webpack.dev.js",
-    "lint": "./node_modules/.bin/eslint . --ext .js,.vue; exit 0",
-    "lintfix": "./node_modules/.bin/eslint . --fix --ext .js,.vue",
-    "lintwin": "./node_modules/.bin/eslint . --ext .js,.vue",
+    "dev": "webpack --watch --progress --hide-modules --config  webpack.dev.js",
+    "lint": "eslint . --ext .js,.vue",
```

Comment:
Small cleanup as noted in the commit message, `--watch` is a little addition enroute to setting up live/hot reload

Reply:
1. 👍
2. nope
3. not too sure what this was for unfortunately (might have been something with node back during the original vue-strap), but our eslint script in `markbind-cli` dosen't have this. Seems to work fine on ci and fails properly when I manually tested `npm run lint && ls`
4. 👍

Reply:
I'll force push later but just this commit will be changed (for the commit message), the rest will be fixup commits